plutus-playground-server/usecases/Vesting.hs

```diff
@@ -205,9 +205,10 @@ retrieveFundsC vesting payment = do
         remainingOutputs = case liveness of
                             Alive -> payIntoContract remainingValue
                             Dead  -> mempty
+        nextSlotTime = TimeSlot.slotToPOSIXTime nextSlot
```

Comment:
I'm going to stop commenting on these: I guess generally every occurrence of one of these conversion functions is a place where we should consider making the underlying function handle times instead of slots.